example/otel-collector/main.go

```diff
@@ -126,6 +187,7 @@ func main() {
 	defer span.End()
 	for i := 0; i < 10; i++ {
 		_, iSpan := tracer.Start(ctx, fmt.Sprintf("Sample-%d", i))
+		runCount.Add(ctx, 5, metric.WithAttributes(commonAttrs...))
```

Comment:
nit: given this is the iteration counter ...

```suggestion
		runCount.Add(ctx, 1, metric.WithAttributes(commonAttrs...))
```